### PR TITLE
Merge develop to main

### PR DESCRIPTION
**Pull request by mama10**

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -1,3 +1,63 @@
...
+  * Made the decoder more robust against invalid Collada tags.
+* USD Decoder:
+  * The USD importer has been improved to accept resolve map keys for textures which represent absolute and relative paths.
+* USD Encoder:
+  * The USD exporter now uses separate indices for all vertex attributes. This allows for "watertight" meshes with hard edges (i.e. a shared point but different vertex normals). 
+  *	The USD exporter has been improved to pass validation checks for default prims, stage up-axis and material binding API
+* GLTF Decoder & Encoder:
+  * The glTF material attribute "doubleSided" is now handled both by the exporter and importer. It is translated to and from the CGA material attribute of same name.   
+* Unreal Encoder:
+  * Does not change the curent working directory anymore when being loaded in in `prt:init()`.
```
Comment (mistafunk) [inline changelog.md:59]: typo - duplicate "in"

**Comment by mistafunk** [comment]
Changelog is formatted a bit inconsistently - some functions are using the `code` formatting, some just "double quotes", some nothing at all...

File: changelog.md (hunk 1 above)
Comment (mama10) [reply changelog.md]: fixed

**Comment by matthias-specht** [comment]
looks good

**Pull request merged**